backend/server.js

```diff
@@ -1,25 +1,28 @@
 const express = require('express');
 const mongoose = require('mongoose');
 const cors = require('cors');
-const authRoutes = require('./routes/authRoutes');
-const subscriptionRoutes = require('./routes/subscription');
+const authRoutes = require('./Routes/authRoutes');
+const userRoutes = require('./Routes/userRoutes');
+const SubscriptionRoute = require('./Routes/subscription')
```

Comment:
you have changed the path of the package with the 'R' in the routes which is still in the small case and will create errors.

Reply:
I did'nt change anything, just re-wrote the paths in server.js. It is not causing any errors